migration/migrating_4_2_4/deploying-cam-4-2-4.adoc

```diff
@@ -1,72 +1,84 @@
 [id='deploying-cam-4-2-4']
-= Deploying the Migration Toolkit for Containers
+= Installing and upgrading the Migration Toolkit for Containers
 include::modules/common-attributes.adoc[]
 :context: migrating-4-2-4
 :migrating-4-2-4:
 
 toc::[]
 
-You can install the {mtc-full} Operator on your {product-title} {product-version} target cluster and 4.2 source cluster. The {mtc-full} Operator installs the {mtc-full} ({mtc-short}) on the target cluster by default.
+You can install the {mtc-full} Operator on your {product-title} {product-version} target cluster and 4.2 source cluster.
 
-[NOTE]
+{mtc-short} is installed on the target cluster by default. You can install {mtc-short} link:https://access.redhat.com/articles/5064151[on an {product-title} 3 cluster or on a remote cluster].
```

Comment:
_You can install the {mtc-full} Operator on your {product-title} {product-version} target cluster_
If the MTCO is installed on the target cluster by default, why do I need instructions to install if on the target cluster? I'm confused here.

Reply:
Because you still have to install it, no matter what cluster it is installed on. It does not install itself. But if you want to isntall the MTC on a different cluster, you have to change the defaults. It may be obvious but we have to call it out because sometimes people assume that you can choose which cluster to install it during the installation. That is not the case. The user has to change the MigController CR manifest if MTC is installed on a non-target cluster.

Reply:
I reworded it to make it clear that it is the migration-controller pod that runs on the target cluster by default.

Reply:
I got it. I do like the original sentence better; it seems more clear to me. 
_The Migration Toolkit for Containers Operator installs the Migration Toolkit for Containers (MTC) on the target cluster by default._
The way you have it sounds like the MTC is already installed by default. The default is *where* the MTC is installed; I read it that it was installed by default.

Reply:
Actually, MTC is installed on all the clusters. The difference is where the migration-controller pod is running.